src/test/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/aktivitet/AktivitetControllerTest.kt

```diff
@@ -62,18 +63,18 @@ class AktivitetControllerTest : IntegrationTest() {
     }
 
     private fun sendOpprettAktivitetRequest(
-        lagreAktivitet: LagreAktivitet,
+        lagreAktivitet: LagreVilkårperiodeNy,
```

Comment:
Kan ev kalle denn for `LagreVilkårperiodeV2` så den matcher endepunktene der den blir brukt